src/services/GroupMeService.test.ts

```diff
@@ -34,7 +34,7 @@ describe("GroupMeService", () => {
 
   describe(".createMentions()", () => {
     it("return a correct mention string and a valid mention object for the string", () => {
-      const [mentionsString, mention] = groupMeService.createMentions(members);
+      const [mentionsString, mention] = groupMeService.createMentions("", members);
```

Comment:
can you make the API

groupMeService(User[], emoji?: string?

Reply:
helps the default

Reply:
opinion welcome

Reply:
added a default